### PR TITLE
Add WRC FAQs to the FAQ section

### PR DESCRIPTION
**Pull request by Laura-O**
This is supposed to add WRC FAQs to the FAQ section.
Details:
* Added 4 regulations questions
* Added one general FAQ (no. 9)
* Changed question 4 of the existing FAQ (added link to translations)

Screenshots:

faq1

faq2

### CODE REVIEW

File: WcaOnRails/config/locales/en.yml [hunk 1]
```diff
@@ -986,7 +986,8 @@ en:
...
         #context: Answer to "What are the requirements for attending a WCA competitions? ..."
         content:
-          '1': "In general, there are not many requirements for a person to participate at a WCA competition. However, all participants of a competition are expected to have a good understanding of the <a href='https://www.worldcubeassociation.org/regulations/'>WCA regulations</a>. In addition, we would recommend having a look at <a target='_blank' href='http://www.cubingusa.com/ctutorial.php'>Cubing USA's competitor tutorial</a>, which gives several nice hints for newcomers."
+          '1': "In general, there are not many requirements for a person to participate at a WCA competition. However, all participants of a competition are expected to have a good understanding of the <a href='https://www.worldcubeassociation.org/regulations/'>WCA Regulations</a>. If English is not your first language and you have trouble to understand the document, please read the <a href='https://www.worldcubeassociation.org/regulations/translations/'>Translations</a>."
```
Comment (jfly) [inline WcaOnRails/config/locales/en.yml]: "trouble to" sounds a bit awkward here, I'd rephrase this as "and you have trouble understanding...".

File: WcaOnRails/config/locales/en.yml [hunk 2]
```diff
@@ -1021,6 +1022,31 @@ en:
         content:
           #context: Yup, nothing to translate here! Just hit "Use the original version"
           '1': "%{current_user_claim}"
+      '9':
+        title: "I am registered for a competition and want to sign up for an additional event, what should I do? Can I bring guests?"
+        #context: Answer to "I want to sign up or an additional event ...?"
```
Comment (jfly) [inline WcaOnRails/config/locales/en.yml]: "sign up or" -> "sign up for"

File: WcaOnRails/config/locales/en.yml [hunk 3]
```diff
@@ -1021,6 +1022,31 @@ en:
...
+        content:
+          '1': "Please contact the organizer. You can find the contact information on the competition website. If you are unsure where to find it, check the <a href='/competitions'>list of competitions</a>."
```
Comment (jfly) [inline WcaOnRails/config/locales/en.yml]: Should we be using `%{comp_search_form}` here? Or maybe link to https://www.worldcubeassociation.org/faq#how-can-i-register-for-a-competition-who-can-i-refer-to-if-i-have-problems-registering-for-a-competition?

File: WcaOnRails/config/locales/en.yml [hunk 4]
```diff
@@ -1021,6 +1022,31 @@ en:
...
+      'regulations1':
+        title: "Is my puzzle competition legal?"
+        #context: Answer to "Is my puzzle competition legal?"
+        content:
+          '1': "You can find the regulations regarding puzzles in <a href='https://www.worldcubeassociation.org/regulations/#3'>Article 3</a>. All puzzles have to be approved by the WCA Delegate of the competition. If you are unsure, please ask him. To be on the safe side, you should bring a replacement puzzle in case you are not allowed to use it."
```
Comment (jfly) [inline WcaOnRails/config/locales/en.yml]: Just a thought about the "him", since the regulations make such an effort to remain gender neutral (https://github.com/thewca/wca-regulations/issues/40 and https://github.com/thewca/wca-regulations/issues/288), maybe we should strive for the same here?

File: WcaOnRails/config/locales/en.yml [hunk 5]
```diff
@@ -1021,6 +1022,31 @@ en:
...
+      'regulations2':
+        title: "May I use a magnetic puzzle in an official WCA competition?"
+        #context: Answer to "May I use a magnetic puzzle in an official WCA competition?"
+        content:
+          '1': "You are allowed to use magnetic puzzles as long as they comply with the <a target='_blank' href='https://www.worldcubeassociation.org/regulations/'>WCA Regulations</a>. Please pay particular attention to <a href='https://www.worldcubeassociation.org/regulations/#3h'>Paragraph 3h</a>."
+      'regulations3':
+        title: "I lost a center cap or a piece of my puzzle. Am I still allowed to use it?"
+        #context: Answer to "Am I allowed to use damaged puzzles?"
+        content:
+          '1': "You are not allowed to use this puzzle. Puzzles must not have any damages or other differences that significantly distinguish any piece from a similar piece (see <a href='https://www.worldcubeassociation.org/regulations/#3j'>Regulation 3j</a>)."
```
Comment (jfly) [inline WcaOnRails/config/locales/en.yml]: nit: Earlier you referred to "paragraph 3h", but here you refer to "regulation 3j". Are these regulations or paragraphs?

File: WcaOnRails/config/locales/en.yml [hunk 6]
```diff
@@ -1021,6 +1022,31 @@ en:
         content:
           #context: Yup, nothing to translate here! Just hit "Use the original version"
           '1': "%{current_user_claim}"
+      '9':
+        title: "I am registered for a competition and want to sign up for an additional event, what should I do? Can I bring guests?"
+        #context: Answer to "I want to sign up or an additional event ...?"
+        content:
+          '1': "Please contact the organizer. You can find the contact information on the competition website. If you are unsure where to find it, check the <a href='/competitions'>list of competitions</a>."
+      'regulations1':
```
Comment (jfly) [inline WcaOnRails/config/locales/en.yml]: I assume that this `regulations1` does what you want here? I just realized I have no idea if there's any guarantee that our FAQs remain in any particular order. @viroulep, the relevant code is [here](https://github.com/jfly/worldcubeassociation.org/blob/546eeb12deb4bc5732674a560bd5c55bf87e476c/WcaOnRails/app/views/static_pages/faq.html.erb#L35), I believe you wrote this, do you know if we should worry about ordering at all?

Comment (viroulep) [inline WcaOnRails/config/locales/en.yml]: I was actually going to drop a message about that when I saw your review!
The key names don't matter, only the order of the keys **in English** does (I mentioned it [here](https://github.com/thewca/worldcubeassociation.org/pull/1097#issue-201022738)).

The `I18n.t` fetches English keys and preserves the order, and the `deep_merge` just override the ones already existing (meaning that even if foreign locales are in the wrong order, they are still displayed in the English order).

Comment (Laura-O) [inline WcaOnRails/config/locales/en.yml]: Ok, so that means the way I did that is fine? :)

Comment (viroulep) [inline WcaOnRails/config/locales/en.yml]: Yes totally!

You can even use "better" key names if you want (eg: to better represent what the question is about), as long as you put them in the order you want them displayed.

File: WcaOnRails/config/locales/en.yml (hunk 3 above)
Comment (Laura-O) [inline WcaOnRails/config/locales/en.yml]: We have [https://www.worldcubeassociation.org/faq#how-can-i-find-a-wca-competition]() which is pretty much the same question. Do you think linking to the FAQ is easier to understand?

**Comment by Laura-O** [comment]
Thanks for the comments and proofreading again @jfly.
I will update the PR when I have a neat solution for the competition list link. 😄

File: WcaOnRails/config/locales/en.yml (hunk 3 above)
Comment (jfly) [inline WcaOnRails/config/locales/en.yml]: :man_shrugging:  No strong preferences here. Whatever you like!

**Comment by viroulep** [comment]
Merged and deployed per Laura's request.
